src/ds/ds-options.cpp

```diff
@@ -597,23 +600,59 @@ namespace librealsense
 
     void emitter_always_on_option::set(float value)
     {
-        command cmd(ds::LASERONCONST);
-        cmd.param1 = static_cast<int>(value);
+        command cmd( _hmc_set_opcode );
+        // reverse logic legacy <> new opcode as we query `LASERONCONST` vs 'APM_STROBE_ON' and return 'EMITTER_ALWAYES_ON'
+        bool always_on = _is_legacy ? value : ( value != 1.0f );
```

Comment:
Value of 1 means **not** always on?

Reply:
Ahh... I understood from your query comment. Here The comment is unclear.

Reply:
Pushed an updated comment